hera_pspec/uvpspec.py

```diff
@@ -853,29 +903,29 @@ def key_to_indices(self, key, omit_flags=False):
         """
         Convert a data key into relevant slice arrays. A data key takes the form
 
-        (spw_integer, ((ant1, ant2), (ant3, ant4)), pol_string)
+        (spw_integer, ((ant1, ant2), (ant3, ant4)), (pol12, pol34))
```

Comment:
small comment: I think `pol12, pol34` is a little confusing b/c throughout the code we adopt `pol1, pol2`, where `pol` is a visibility polarization, not a feed polarization. So I'd stick to the convention that `pol` refers to visibility not feed polarization, such that this reads `(pol1, pol2)`.